lsp-mode.el

```diff
@@ -4043,7 +4048,8 @@ interface TextDocumentEdit {
                 (mkdir (f-dirname file-name) t)
                 (f-touch file-name)
                 (when (lsp:create-file-options-overwrite? options?)
-                  (f-write-text "" nil file-name))))
+                  (f-write-text "" nil file-name))
+                (find-file-noselect file-name)))
```

Comment:
why this is needed?

Reply:
For some reason, without this, when user access the buffer, emacs prompt with `File foo changed on disk. Discard your edits? (y or n)`, if user types `y`, the buffer becomes blank, if user types `n` then buffer becomes with the changes of those edits correctly.
I realized that adding this makes emacs load the buffer file after creating the file which makes sense